build.zig

```diff
@@ -8,7 +8,10 @@ pub fn build(b: *std.Build) void {
     });
 
     lib.linkLibC();
-    lib.addCSourceFile(.{ .file = .{ .path = "lib/src/lib.c" }, .flags = &.{} });
+    lib.addCSourceFiles(&.{
+        "lib/src/lib.c",
+    }, &.{"-std=gnu11"});
+    lib.defineCMacroRaw("DTREE_SITTER_REUSE_ALLOCATOR");
```

Comment:
Same question here - I think `TREE_SITTER_REUSSE_ALLOCATOR`  should only be needed when compiling an external scanner.